tests/test_config.py

```diff
@@ -20,10 +20,10 @@
 
 class ConfigTest(unittest.TestCase):
 
-    def test_temporary_config(self):
+    def test_temporary_config(self) -> None:
         """Check that using a temporary config has a temporary effect."""
         original_value = GlobalConfig.delay_before_drop
-        new_value = original_value * 10
+        new_value = int(original_value) * 10
```

Comment:
This can modify behavior and I cannot confirm what it does

Reply:
Done